docs/learn-collator.md

```diff
@@ -34,13 +34,13 @@ them to send messages from parachain A to parachain B.
 
 ### Collators in the Wild
 
-Collators are being used on the [Rococo](build-parachains-rococo) testnet, and you can learn more
-about how they are used with Cumulus via the [Cumulus](https://github.com/paritytech/cumulus/)
-repository.
+Collators are being used on many production and testing relay chains now!
```

Comment:
What are "testing relay chains"? Also no `!` please unless absolutely necessary.

Reply:
Yeah, it's a weak statement, I agree. I have updated to what I hope is more acceptable